.github/PULL_REQUEST_TEMPLATE/console_pr.md

```diff
@@ -0,0 +1,14 @@
+**CONSOLE Fixes**: 
+<!-- For e.g Fixes: https://issues.redhat.com/browse/CONSOLE-XXXX -->
+
+**Solution Description**: 
+<!-- Describe your code changes in detail and explain the solution / functionality -->
+
+**Screen shots / gifs / design review**: 
+<!-- Add screenshots/gifs for UI changes. If change requires a UX review, tag @openshift/team-ux-review -->
+
+**Add a code reviewer assignee**: 
+<!-- @ someone to review this pr -->
+
+**Test cases:**
+<!-- Outline any test cases here -->
```

Comment:
Not sure if the template should contain the direct assignees, since they might change.
My concern is that this way they could get notification on every PR.
Would suggest to have a template, something like:
```
QE Approver: 
/assign 
Docs Approver: 
/assign
PX Approver: 
/assign
Console Approver: 
/assign 
```

Reply:
Ok, kept the updates suggested and removed the hard coded assignees